bigquery/tests/unit/test_client.py

```diff
@@ -1320,6 +1322,38 @@ def test_get_table(self):
         conn.api_request.assert_called_once_with(method="GET", path="/%s" % path)
         self.assertEqual(table.table_id, self.TABLE_ID)
 
+    def test_get_table_sets_user_agent(self):
+        creds = _make_credentials()
+        http = mock.create_autospec(requests.Session)
+        mock_response = http.request(
+            url=mock.ANY, method=mock.ANY, headers=mock.ANY, data=mock.ANY
+        )
+        http.reset_mock()
+        mock_response.status_code = 200
+        mock_response.json.return_value = self._make_table_resource()
+        user_agent_override = client_info.ClientInfo(user_agent="my-application/1.2.3")
+        client = self._make_one(
+            project=self.PROJECT,
+            credentials=creds,
+            client_info=user_agent_override,
+            _http=http,
+        )
+
+        table = client.get_table(self.TABLE_REF)
```

Comment:
[Lint failure here](https://source.cloud.google.com/results/invocations/51249904-7b66-49db-8836-7ace0e594049/targets/cloud-devrel%2Fclient-libraries%2Fgoogle-cloud-python%2Fpresubmit%2Fbigquery/log):  the `table` variable is never used.  You can just invoke `client.get_table` without binding the return value, or use `_`.